### DISCUSSION TITLE
Hello, good morning. Does anyone know how I could set the word limit per sentence to 1? I'm quite lost in this :c

### DISCUSSION THREAD

**Discussion by superjuan5** [absadiki/subsai#32] [Q&A] [answered]
Hello, good morning. Does anyone know how I could set the word limit per sentence to 1? I'm quite lost in this :c And the truth is, it would be very helpful for me to know how to limit the maximum number of words or characters per sentence since I want to optimize the editing time of TikToks :c

**Comment by absadiki** [comment, marked as answer]
Hello @superjuan5,

I think I already answered a similar question [here](https://github.com/abdeladim-s/subsai/discussions/29#discussioncomment-5797601). 

Let me know if that solved your issue ?

**Reply by superjuan5** [reply]
Yep :D , but how can i use my GPU to render the subtitles?

**Reply by absadiki** [reply]
Sorry, I didn't understand your question ? Could you please elaborate more ?

**Reply by superjuan5** [reply]
![image](https://github.com/abdeladim-s/subsai/assets/69076369/b689c6a1-963a-49a1-8b2a-5a34a2a823cb)
How can i use my GPU in this whisper model?

**Reply by absadiki** [reply]
oh, I understand, `whisper.cpp` is CPU only, (I think there is a GPU support but it is not working on Windows, so as of now you can only use CPU).
Unfortunately it is the only model supporting the feature you are looking for!

**Reply by superjuan5** [reply]
ou :c , well , thank you abdeladim :D

**Reply by absadiki** [reply]
You are welcome @superjuan5 :D
